### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by anselmbradford**
General code cleanup.

## Changes

- Fixes issue with missing `breakpointPx` variable (mistake from https://github.com/cfpb/cfgov-refresh/pull/711)
- Updates wrong code comment.
- Removes extraneous gitignore entry.

## Review

- @jimmynotjim 
- @KimberlyMunoz 
- @sebworks

### CODE REVIEW

File: src/static/js/routes/the-bureau/index.js [hunk 1]
```diff
@@ -16,8 +16,8 @@ function init() {
     return;
   }
 
-  // MobileCarousel with 599px breakpoint.
-  var mobileCarousel = new MobileCarousel( 599 );
+  var breakpointPx = 599;
```
Comment (sebworks) [inline src/static/js/routes/the-bureau/index.js]: Should this be in a config file?

File: src/static/css/cf-enhancements.less [hunk 2]
```diff
@@ -968,7 +968,7 @@ textarea.input__long {
   name: Hide only when JS is active
   family: cf-core
   patterns:
-    - name: .u-reset
```
Comment (KimberlyMunoz) [inline src/static/css/cf-enhancements.less]: Why this change? Seems inconsistent with the comments and css below.

File: the-bureau/about-rich-cordray/index.html [hunk 3]
```diff
@@ -52,7 +52,7 @@
                 <div class="media">
                     <div class="media_image-container
                                 media_image-container__wide-margin">
-                        <img src="/static/img/cordray_round_90.jpg" alt="Portrait of Director Richard Cordray" width="60">
```
Comment (KimberlyMunoz) [inline the-bureau/about-rich-cordray/index.html]: Could we just remove the `width`?

File: src/static/css/cf-enhancements.less (hunk 2 above)
Comment (anselmbradford) [inline src/static/css/cf-enhancements.less]: `.u-reset` is the block [before this one](https://github.com/cfpb/cfgov-refresh/blob/cleanup/src/static/css/cf-enhancements.less#L947-L965). `.js-is-hidden` is [this block](https://github.com/cfpb/cfgov-refresh/blob/cleanup/src/static/css/cf-enhancements.less#L967-L986).

Comment (KimberlyMunoz) [inline src/static/css/cf-enhancements.less]: Blame it on the lack of  :poultry_leg:. Got it.

**Comment by anselmbradford** [comment]
I've removed the image size changes and opened https://github.com/cfpb/cfgov-refresh/issues/722 ... digging into this I realized it's a bigger issue than this PR can encompass.

**Comment by anselmbradford** [comment]
Added edit in https://github.com/cfpb/cfgov-refresh/commit/b4a88894e062b30546661940104a19ab2f480377 to remove extraneous entry in the gitignore file.

Can I get another :+1: ?

**Comment by sebworks** [comment]
:+1:

File: src/static/js/routes/the-bureau/index.js (hunk 1 above)
Comment (anselmbradford) [inline src/static/js/routes/the-bureau/index.js]: As mentioned in convo, sounds like a good idea, but out of scope with this PR. Opened https://github.com/cfpb/cfgov-refresh/issues/726
